pkg/humio/client.go

```diff
@@ -11,19 +11,26 @@ import (
 	"strings"
 
 	"github.com/grafana/grafana-plugin-sdk-go/backend/httpclient"
-	"github.com/shurcooL/graphql"
+	"github.com/hasura/go-graphql-client"
 )
 
 type Client struct {
 	URL        *url.URL
 	HTTPClient *http.Client
+	Auth
 }
 
 type Config struct {
 	Address *url.URL
 	Token   string
 }
 
+type Auth struct {
+	OAuthPassThru bool
+	AccessToken   string
```

Comment:
IMO this property could be better named to reflect it's type.

- `OAuthPassThruEnabled`
- `HasOAuthPassThru`
- `WithOAuthPassThru`

Reply:
Gonna stick with this naming because of Grafana convention